.env

```diff
@@ -16,3 +16,5 @@ DJANGO_DEBUG=0
 
 # Override this in .private.env
 SECRET_KEY=MyAwesomeSecretKey
+
+CERTIFICATE_DB=/var/taskd/certificates.sqlite3
```

Comment:
I'm _pretty_ sure this is the wrong choice -- that path isn't on a mounted volume, but `/var/taskd/orgs/certificates.sqlite3` (https://github.com/coddingtonbear/inthe.am/blob/fceb654e58bf7e16752bdc63ee1e075b59df8c8b/docker/taskd/Dockerfile#L5 & https://github.com/coddingtonbear/inthe.am/blob/e1fd8781dd80571975b3a424438323464b663903/docker-compose.yml#L139) is.  I haven't checked thoroughly, but I believe that if we use this path, the certificates database will be lost every time the node is restarted, no?

Reply:
Hi, yes, you were right! The problem was on the other way, in `taskd_services/http/api.py`

```python
TASKD_DATA = os.environ["TASKDDATA"]
...
CERT_DB_PATH = os.environ.get(
    "CERT_DB_PATH", os.path.join(TASKD_DATA, "certificates.sqlite3")
)
```
`CERT_DB_PATH` had `/var/taskd/certificates.sqlite3` instead of the correct one.  Here we have two options: 1) in the default value join `orgs` as well:
```python
CERT_DB_PATH = os.environ.get(
    "CERT_DB_PATH", os.path.join(TASKD_DATA, "orgs", "certificates.sqlite3")
)
```
Or 2) define `CERT_DB_PATH` somewhere (Dockerfile or .env file).

In my opinion, it's better 1) to maintain default paths, what do you think?

Reply:
Ooo -- that's totally the source of a weird mystery I was trying to investigate some months ago!  I've checked, and we're definitely writing that certificates database to something that is not saved to that volume, and it's causing a number of interesting bugs.

The right choice is of course for us to not calculate `CERT_DB_PATH` in `taskd/services/http/api.py` and to instead just have it read from the environment variable `CERTIFICATE_DB`, but I can sort that out for you, I think.